app/actions/websocket/index.ts

```diff
@@ -247,40 +255,41 @@ export async function handleEvent(serverUrl: string, msg: WebSocketMessage) {
             break;
 
         case WebsocketEvents.CHANNEL_CREATED:
+            handleChannelCreatedEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelCreatedEvent(msg));
         case WebsocketEvents.CHANNEL_DELETED:
             handleChannelDeletedEvent(serverUrl, msg);
             break;
         case WebsocketEvents.CHANNEL_UNARCHIVED:
+            handleChannelUnarchiveEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelUnarchiveEvent(msg));
         case WebsocketEvents.CHANNEL_UPDATED:
+            handleChannelUpdatedEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelUpdatedEvent(msg));
         case WebsocketEvents.CHANNEL_CONVERTED:
+            // covered by CHANNEL_UPDATED
+            // handleChannelConvertedEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelConvertedEvent(msg));
         case WebsocketEvents.CHANNEL_VIEWED:
+            handleChannelViewedEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelViewedEvent(msg));
         case WebsocketEvents.CHANNEL_MEMBER_UPDATED:
+            handleChannelMemberUpdatedEvent(serverUrl, msg);
             break;
 
-        // return dispatch(handleChannelMemberUpdatedEvent(msg));
-        case WebsocketEvents.CHANNEL_SCHEME_UPDATED:
-            break;
+            // case WebsocketEvents.CHANNEL_SCHEME_UPDATED:
+            //     handleChannelSchemeUpdatedEvent(serverUrl, msg);
+            //     break;
```

Comment:
After looking into the behaviour it seems that if we change the scheme and it gets created or deleted (because it is equal or different from the higher level scheme), we receive a channel updated event to change the scheme.

Therefore, seems save to remove.